README.md

```diff
@@ -40,6 +40,7 @@ Please contact sales@flexera.com to learn more.
 - [AWS Reserved Instances Report by Billing Center](./cost/aws/reserved_instances/report_by_bc)
 - [AWS Reserved Instance Recommendations](./cost/aws/reserved_instances/recommendations)
 - [AWS Savings Plan Recommendations](./cost/aws/savings_plan/recommendations)
+- [AWS Delete Old Snapshots](./cost/aws/old_snapshots/)
```

Comment:
This should be under storage.  also please rename to AWS Old Snapshots

Reply:
Moved to storage